nototools/hb_input.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+"""Provides a functions to generate harbuzz input.
+
+The input is returned as a list of strings, suitable for passing into
+subprocess.call or something similar.
+"""
+
+
+from fontTools.ttLib import TTFont
+
+from nototools import summary
+
+
+def all_hb_inputs(path, output_dir):
+    """Generate harfbuzz inputs for all glyphs in a given font."""
+
+    inputs = []
+    font = TTFont(path)
+    reverse_cmap = build_reverse_cmap(font)
+    glyph_names = font.getGlyphOrder()
+    for name in glyph_names:
+        inputs.append([path] + hb_input_from_name(font, name, reverse_cmap))
+    return inputs
+
+
+def build_reverse_cmap(font):
```

Comment:
This assumes a 1-1 cmap.  Might warn, at least, when it's not.  As far as generating some kind of input, though, it doesn't matter I suppose.